boot-mongodb-elasticsearch/src/test/java/com/example/mongoes/web/controller/RestaurantControllerIntTest.java

```diff
@@ -26,12 +27,29 @@ void createRestaurant() {
                 .expectStatus()
                 .isCreated()
                 .expectHeader()
-                .contentType(MediaType.APPLICATION_JSON)
-                .expectHeader()
-                .exists("location")
+                .exists(HttpHeaders.LOCATION);
+    }
+
+    @Test
+    void findRestaurantByName_WithNonExistentName_ShouldReturnNotFound() {
```

Comment:
@coderabbitai, can you please helpe me with writing Integration tests for all endpoints in RestaurantController

Reply:
@coderabbitai, somehow the values are not getting picked from elasticsearch when called . Can you please help me to fix.